setup.py

```diff
@@ -3,8 +3,8 @@
 setup(
     name="noma",
     version="0.5.1",
-    packages=["noma"],
-    install_requires=["psutil", "docopt", "requests", "docker-compose"],
+    packages=["noma", "configobj"],
```

Comment:
What's that?  Why does it add 2k LOC to the repo here?

Reply:
I had to make some changes to the way the library writes out ini files to conform to `bitcoind` and `lnd` style.

I'll submit a PR upstream, once it's merged and released `configobj/` can be removed

Reply:
Cool, just add a comment saying that somewhere :).

Ideally with a link to an issue tracking it.